### PR TITLE
Chore: Implementing animation, to show the content.

### PR DESCRIPTION
**Pull request by Junior331**
## Description

Implementando animação de mostrar conteúdo de acordo com o scroll.

## Developer Checks

- [x] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [x] Tested on browser device
- [x] Tested on Mobile device

### CODE REVIEW

**Comment by jonataspinto** [comment]
Deu ruim no build

@Junior331

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.69 | ✅ |
| First Contentful Paint | 1.9 s |  |
| Time to Interactive | 5.5 s |  |
| Speed Index | 3.2 s |  |
| Total Blocking Time | 1,380 ms |  |
| Largest Contentful Paint | 2.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.89 | ✅ |
| First Contentful Paint | 2.4 s |  |
| Time to Interactive | 6.2 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 110 ms |  |
| Largest Contentful Paint | 2.7 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by Junior331** [comment]
> Deu ruim no build
> 
> @Junior331

Resolvido.

**Review by jonataspinto — CHANGES_REQUESTED** [review]
A seção de nossos resultados ficou com um pequeno bug no mobile. Vidas transformadas não aparece

File: src/mocks/DescriptionProfile.mock.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
-type studentSectionProfile = {
+export type studentSectionProfile = {
   title: string;
   description: string;
+  dataAos: string;
```
Comment (jonataspinto) [inline src/mocks/DescriptionProfile.mock.ts:4]: A ideia desses mocks é deixar isolado toda parte de dados, pensando em controlar vida CMS um dia. Acho que esses dados relacionados ao `aos` não cabem aqui.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
-    <HowWeDo {...howWeDoMock} />
-    <StudentProfile {...studentProfileSection} />
-    <SelectiveProcess {...scheduleMock} />
-    <Subscribe />
-    <OurResults />
-    <Footer />
-  </Layout>
-);
+const Home = () => {
+  useEffect(() => {
```
Comment (jonataspinto) [inline src/pages/index.tsx:26]: Podemos implementar um hook para isso, o que acha @Junior331 ?

**Review by jonataspinto — CHANGES_REQUESTED** [review]

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.71 | ✅ |
| First Contentful Paint | 1.7 s |  |
| Time to Interactive | 7.5 s |  |
| Speed Index | 2.9 s |  |
| Total Blocking Time | 890 ms |  |
| Largest Contentful Paint | 2.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.79 | ✅ |
| First Contentful Paint | 2.6 s |  |
| Time to Interactive | 7.2 s |  |
| Speed Index | 4.1 s |  |
| Total Blocking Time | 210 ms |  |
| Largest Contentful Paint | 3.2 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.94 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 6.3 s |  |
| Speed Index | 1.7 s |  |
| Total Blocking Time | 160 ms |  |
| Largest Contentful Paint | 1.8 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.87 | ✅ |
| First Contentful Paint | 2.3 s |  |
| Time to Interactive | 6.5 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.91 | ✅ |
| First Contentful Paint | 1.4 s |  |
| Time to Interactive | 6.6 s |  |
| Speed Index | 2.0 s |  |
| Total Blocking Time | 250 ms |  |
| Largest Contentful Paint | 1.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.82 | ✅ |
| First Contentful Paint | 2.6 s |  |
| Time to Interactive | 7.0 s |  |
| Speed Index | 4.0 s |  |
| Total Blocking Time | 160 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Pull request merged**